build.sh

```diff
@@ -50,6 +50,10 @@ patch -i "$DIR/Dockerfile.patch" "$SRC/docker/amd64/$DB_TYPE/Dockerfile" -o "$DI
 sed -E "s/(FROM[[:space:]]*rust:)[^[:space:]]+(.+)/\1${OS_VERSION_NAME}\2/g" -i "$DIR/Dockerfile"
 sed -E "s/(FROM[[:space:]]*debian:)[^-]+(-.+)/\1${OS_VERSION_NAME}\2/g" -i "$DIR/Dockerfile"
 
+# Prepare Controlfile
+CONTROL="$DIR/debian/control"
+sed -i "s/Version:.*/Version: $REF-1/" "$CONTROL"
```

Comment:
As this control file is in git tree, I'd prefer to have a template (and ignore the generated file).

Reply:
So you think of something like `Version: -` as a template?

Reply:
I mean have a `control.dist` file and use `sed` the way you do to produce the `control` file (which would be added to gitignore)